clients/ops/admin-ui/src/features/privacy-requests/constants.ts

```diff
@@ -10,4 +10,5 @@ export const SubjectRequestStatusMap = new Map<string, string>([
   ["In Progress", "in_processing"],
   ["New", "pending"],
   ["Paused", "paused"],
+  ["Unverified", "identity_unverified"],
```

Comment:
Thanks for adding these frontend elements

Reply:
I didn't have a choice! If a privacy request exists in the database, and it had this new request status, you'd get a UI error on the privacy request page that made it unusable.
![Screen Shot 2022-08-22 at 12 35 35 PM](https://user-images.githubusercontent.com/9755598/186188029-7432634a-119f-4c94-9c73-cd7c59457434.png)